tests/callbacks/test_mlperf_callback.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import numpy as np
+import pytest
+from torch.utils.data import DataLoader
+
+from composer import Trainer
+from composer.callbacks import MLPerfCallback
+from tests.common import RandomClassificationDataset, SimpleModel
+
+logging.basicConfig(filename="/Users/hanlintang/composer/package_checker.log", level=logging.INFO)
+logging.getLogger().addHandler(logging.StreamHandler())
+formatter = logging.Formatter("%(levelname)s - %(message)s")
+logging.getLogger().handlers[0].setFormatter(formatter)
+logging.getLogger().handlers[1].setFormatter(formatter)
+
+
+@pytest.fixture
+def config():
+    """Returns the reference config."""
+
+    return {
+        'model': SimpleModel(),
+        'train_dataloader': DataLoader(
+            dataset=RandomClassificationDataset(),
+            batch_size=4,
+            shuffle=False,
+        ),
+        'eval_dataloader': DataLoader(
+            dataset=RandomClassificationDataset(),
+            shuffle=False,
+        ),
+        'max_duration': '2ep',
+        'deterministic_mode': True,  # testing equivalence
+        'loggers': [],  # no progress bar
+        'callbacks': []
+    }
+
+
+@pytest.mark.filterwarnings(
+    "ignore::DeprecationWarning",)
+def test_mlperf_callback(config, tmpdir):
+    tmpdir = 'mlperf_results'
+    pytest.importorskip("mlperf_logging")
+
+    for run in range(5):
```

Comment:
curious about why 5 runs?

Reply:
To pass the compliance checker, we need to submit 5 consecutive runs of the system. See MLcommons Benchmark rules [here](https://github.com/mlcommons/training_policies/blob/master/training_rules.adoc#benchmark-results).